app/controllers/services_controller.rb

```diff
@@ -141,23 +141,57 @@ def site_code
     params[:site_id] ? Site.find_by(id: params[:site_id]).site_code : "sfsg"
   end
 
-  def eligibility_name
-    params[:eligibility_id] ? Eligibility.find_by(id: params[:eligibility_id].to_i).name : ""
+  def eligibility_names
+    eligibilities = params[:eligibility_id].split(",")
+    Eligibility.where(["id in (?)", eligibilities]).map(&:name)
```

Comment:
Can this line be simplified to this?

```suggestion
    Eligibility.where(id: eligibilities).map(&:name)
```

Reply:
Looks like that does work with multiples too in my postman client! I'll update.